pkg/k8s/apis/cilium.io/v2alpha1/register.go

```diff
@@ -52,6 +52,20 @@ const (
 
 	// CENPName is the full name of Cilium Egress NAT Policy
 	CENPName = CENPPluralName + "." + CustomResourceDefinitionGroup
+
```

Comment:
Should the schema version be bumped in line 37?

Reply:
Not really because it's a new CRD, if it was modifying an existing one then yes.